README.md

```diff
@@ -75,8 +75,9 @@ More information about inventories can be found in the [ansible documentation](h
       * milliner.baseUrl `inventory/production/group_vars/karaf.yml`
   1. Any other variable changes you wish.
 1. Install the roles using `ansible-galaxy`: `$ ansible-galaxy install -r requirements.yml`
-1. Provision the server with `$ ansible-playbook -i inventory/production`
-  - If the host you are provisioning is a Ubuntu 16.04 machine, you may wish to have the playbook install Python for you. This is a requirement to run the playbook. You can do this by passing an additional variable on the command line like this. `$ ansible-playbook -i inventory/production -e "islandora_distro=ubuntu/xenial64"`
+1. Provision the server:
+- If the host you are provisioning is a Ubuntu 16.04 machine, you may wish to have the playbook install Python for you. This is a requirement to run the playbook. You can do this by passing an additional variable on the command line like this. `$ ansible-playbook -i inventory/production playbook.yml -e "islandora_distro=ubuntu/xenial64"`
```

Comment:
@dannylamb @jonathangreen @Natkeeran I always stick the `-u ubuntu` on this lines (and vagrant on the centos one). But is that a requirement, should we be suggesting it?

Reply:
@whikloj ansible_user can be specified in the hosts as well.  Would the ubuntu user always exist?

Reply:
Fair point 👍